plutus-pab/src/Plutus/PAB/Arbitrary.hs

```diff
@@ -211,6 +196,9 @@ instance Arbitrary Slot where
     shrink = genericShrink
 
 instance Arbitrary TxId where
+    arbitrary = pure ""
```

Comment:
We can't reuse the cardano's instance?

If not — maybe return some constant? The genesis TxId for example?